foundational-cxx-support-matrix.md

```diff
@@ -3,15 +3,15 @@
 | Dimension       | Supported Version      | Last Changed | Next Change [^next-change] |
 |-----------------|------------------------|--------------|-------------|
 | C++ Version     | >= 14                  | 2022-07-01   | 2024-12-15  |
-| CMake           | >= 3.13                | 2023-06-01   | 2024-07-01 [^cmake] |
+| CMake           | >= 3.18                | 2024-07-01   | 2024-11-22 [^cmake] |
```

Comment:
I think we can only go to CMake 3.16.  Ubuntu 20.04 is still supported and ships with CMake 3.16:

https://repology.org/project/cmake/versions